### PR TITLE
[Training Manual] 1/3 Chaper 8 (Raster)

### PR DESCRIPTION
**Pull request by ghtmtt**
First part of raster manual updating. No major changes

### CODE REVIEW

File: source/docs/training_manual/rasters/data_manipulation.rst [hunk 1]
```diff
@@ -64,45 +52,39 @@ one (composite) image, right?
...
-* Click :guilabel:`Close` on the :guilabel:`Build Virtual Raster (Catalog)`
-  dialog.  (Don't click :guilabel:`OK` again, otherwise it's going to start
-  running that command again.)
-* You can now remove the original four rasters from the :guilabel:`Layers
-  list`.
-* If necessary, click and drag the new :guilabel:`aerial_photos` raster catalog
-  layer to the bottom of the :guilabel:`Layers` panel so that the other
-  activated layers become visible.
+.. note::  |hard| Keep in mind that you can copy and paste the text in the
+    ``OSGEO Shell`` (Windows user) or ``Terminal`` (Linux and OSX users) to run
```
Comment (DelazJ) [inline source/docs/training_manual/rasters/data_manipulation.rst]: is it not runnable from any other command line shell than OSGeo (btw the spelling!)?

File: source/docs/training_manual/rasters/data_manipulation.rst [hunk 2]
```diff
@@ -64,45 +52,40 @@ one (composite) image, right?
...
+   :guilabel:`Select All` button;
+#. Uncheck the :guilabel:`Place each input file into a separate band` parameter-
+   Notice the text field below. What this dialog is actually doing is that it's
+   writing that text for you. It's a long command that QGIS is going to run.
 
-Notice the text field below. What this dialog is actually doing is that it's
-writing that text for you. It's a long command that QGIS is going to run.
+   .. note:: Keep in mind that you can copy and paste the text in the
+       ``OSGeo Shell`` (Windows user) or ``Terminal`` (Linux and OSX users) to run
+       the command. You can also create script with each GDAL command. This is very
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/training_manual/rasters/data_manipulation.rst]: I am not sure if this is what you meant.  Either `a script` or `scripts`. If `scripts`, `each` should probably be removed.

File: source/docs/training_manual/rasters/data_manipulation.rst [hunk 3]
```diff
@@ -64,45 +52,40 @@ one (composite) image, right?
...
+       handy when the procedure is taking a long time or when you want to schedule
+       specific tasks. Refer always to the :guilabel:`Help` button to get more help
```
Comment (havatv) [inline source/docs/training_manual/rasters/data_manipulation.rst]:
```suggestion
        specific tasks. Use the :guilabel:`Help` button to get more help
```

File: source/docs/training_manual/rasters/data_manipulation.rst [hunk 4]
```diff
@@ -64,45 +52,40 @@ one (composite) image, right?
...
+       on the syntax GDAL commands.
```
Comment (havatv) [inline source/docs/training_manual/rasters/data_manipulation.rst]:
```suggestion
        on the syntax of GDAL commands.
```

File: source/docs/training_manual/rasters/data_manipulation.rst [hunk 5]
```diff
@@ -64,45 +52,40 @@ one (composite) image, right?
...
 
-.. note::  |hard| Keep in mind that the command text is editable, so you can
-   customize the command further if preferred. Search online for the initial
-   command (in this case, :kbd:`gdalbuildvrt`) for help on the syntax.
+#. Finally click on :guilabel:`Run`.
 
-* Click :guilabel:`OK` to run the command.
+.. note:: As you know from the previous modules, :guilabel:`Processing` creates
+    temporary layers by default. If you want you can click on the |browseButton|
```
Comment (havatv) [inline source/docs/training_manual/rasters/data_manipulation.rst]:
```suggestion
     temporary layers by default. To save the file, click on the |browseButton|.
```

File: source/docs/training_manual/rasters/data_manipulation.rst [hunk 6]
```diff
@@ -64,45 +52,40 @@ one (composite) image, right?
...
+    button to save the file on the disk.
```
Comment (havatv) [inline source/docs/training_manual/rasters/data_manipulation.rst]:
```suggestion
     
```

File: source/docs/training_manual/rasters/data_manipulation.rst [hunk 7]
```diff
@@ -64,45 +52,40 @@ one (composite) image, right?
...
 
 .. image:: img/build_virtual_raster.png
    :align: center
 
+You can now remove the original four rasters from the :guilabel:`Layers` Panel
+and leave only the output virtual catalog rater.
```
Comment (havatv) [inline source/docs/training_manual/rasters/data_manipulation.rst]:
```suggestion
 and leave only the output virtual catalog raster.
```

File: source/docs/training_manual/rasters/data_manipulation.rst [hunk 8]
```diff
@@ -116,29 +99,38 @@ rasters in a map, but it may take some time to set up initially.
 Reprojecting rasters
 ...............................................................................
 
-* Click on the menu item :menuselection:`Raster --> Projections --> Warp
-  (Reproject)`.
+Open :guilabel:`Warp (reproject)` from
+:menuselection:`GDAL --> Raster projections`.
+
+You can also reproject virtual rasters (catalogs), as well as enabling a
```
Comment (havatv) [inline source/docs/training_manual/rasters/data_manipulation.rst]:
```suggestion
 You can also reproject virtual rasters (catalogs), enable
```

File: source/docs/training_manual/rasters/data_manipulation.rst [hunk 9]
```diff
@@ -116,29 +99,38 @@ rasters in a map, but it may take some time to set up initially.
...
+multithreaded processing mode and many other options are available.
```
Comment (havatv) [inline source/docs/training_manual/rasters/data_manipulation.rst]:
```suggestion
 multithreaded processing and many more.
```

File: source/docs/training_manual/rasters/data_manipulation.rst [hunk 10]
```diff
@@ -116,29 +99,38 @@ rasters in a map, but it may take some time to set up initially.
...
+:menuselection:`GDAL --> Raster miscellaneous` menu.
+
+As we did for the :ref:`Create a Virtual raster <tm_virtual_raster>` you can use
+the |browseButton| to choose which layers you want to merge.
 
-You can choose to process entire directories instead of single files, giving
-you a very useful built-in batch processing capability. You can specify a
-virtual raster as input file, too, and all of the rasters that it consists of
-will be processed.
+You can merge Virtual raster as input file too, and all of the rasters that it
```
Comment (havatv) [inline source/docs/training_manual/rasters/data_manipulation.rst]:
```suggestion
 You can also specify a Virtual raster as input, and then all of the rasters that it
```

File: source/docs/training_manual/rasters/data_manipulation.rst [hunk 11]
```diff
@@ -116,29 +99,38 @@ rasters in a map, but it may take some time to set up initially.
...
+consists of will be processed.
 
-You can also add your own command line options using the :guilabel:`Creation
-Options` checkbox and list. This only applies if you have knowledge of the GDAL
-library's operation.
+You can also add your own options by opening the :guilabel:`Advanced parameters`
```
Comment (havatv) [inline source/docs/training_manual/rasters/data_manipulation.rst]:
```suggestion
 If you know the GDAL library, you can also add your own options by opening the :guilabel:`Advanced parameters`
```

File: source/docs/training_manual/rasters/data_manipulation.rst [hunk 12]
```diff
@@ -116,29 +99,38 @@ rasters in a map, but it may take some time to set up initially.
...
+menu, if you know the GDAL library.
```
Comment (havatv) [inline source/docs/training_manual/rasters/data_manipulation.rst]:
```suggestion
 menu.
```

**Comment by havatv** [comment]
@DelazJ and @ghtmtt, can the remaining issues be solved?

**Comment by havatv** [comment]
Are we in a deadlock, @DelazJ and @ghtmtt ?

**Comment by ghtmtt** [comment]
@havatv and @DelazJ here my final commit. As for #3202 I'm going to merge this if Travis is happy. We can always change things later on.

**Comment by ghtmtt** [comment]
Again thanks @havatv and @DelazJ for the review!

**Pull request merged**